job-dsls/jobs/FlowJob.groovy

```diff
@@ -405,8 +409,36 @@ mvn -B -e org.sonatype.plugins:nexus-staging-maven-plugin:1.6.5:deploy-staged-re
  -DrepositoryDirectory=$deployDir -s $SETTINGS_XML_FILE -DstagingProfileId=15c3321d12936e -DstagingDescription="kie $kieVersion" -DstagingProgressTimeoutMinutes=40
 # creates a file (list) of the last commit hash of each repository as handover for production
 ./droolsjbpm-build-bootstrap/script/git-all.sh log -1 --pretty=oneline >> git-commit-hashes.txt
-echo $kieVersion > $WORKSPACE/version.txt'''
-
+echo $kieVersion > $WORKSPACE/version.txt
+# creates JSON file for prod
+# resultant sed extraction files
+./droolsjbpm-build-bootstrap/script/git-all.sh log -1 --format=%H  >> sedExtraction_1.txt
+sed -e '1d;2d' -e '/Total/d' -e '/====/d' -e 's/Repository: //g' -e 's/^/\\"/; s/$/\\"/;' -e '/""/d' sedExtraction_1.txt >> sedExtraction_2.txt
+sed -e '$!N;s/\\n/ /g' -e 's/ / : /g' -e 's/$/ ,/' -e '$ s/.$//' sedExtraction_2.txt >> sedExtraction_3.txt
+cat sedExtraction_3.txt
+fileToWrite=$reportDate.json
+commitHash=$(cat sedExtraction_3.txt)
+cat <<EOF > int.json
+{
+   "handover" : {
+   "cut_off_date" : "$cutOffDate",
+   "report_date": "$reportDate",
+   "repos" : [
+      {
+         $commitHash
+      }
+    ],
+   "source_product_tag":"$sourceProductTag",
+   "target_product_build":"$targetProductBuild" 
+   }
+}
+EOF
```

Comment:
Just thinking out loud: what about having this as a template file and then let the scripts fill in the variable content like the dates, tags, etc. WDYT?